Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Build Stage
```

Comment:
This would be the correct way to build an AFL fuzzing image, but this is not necessary for Mayhem. Given that AFLplusplus is a large base image, is it possible to package with vanilla ubuntu instead? You can just provide the path to the afl binary in the Mayhemfile.